docs/docker_container.md

```diff
@@ -239,6 +239,8 @@ Configuration options for server are defined only via command line options and d
 
 ### Cloud Storage Requirements<a name="storage"></a>:
 
+In general OVMS requires "read" and "list" permissions on the model storage side.
```

Comment:
Some short intro could be added here:
OVMS supports a range of cloud storage types. In general OVMS requires "read" and "list" permissions on the [model repository](model_repository.md) side.
Below are specific steps for every storage provider: